trade-finance-manager-ui/templates/case/activity/activity.njk

```diff
@@ -2,11 +2,29 @@
 {% import './_macros/deal-submitted.njk' as dealSubmitted %}
 {% import './_macros/activity-comment-filter.njk' as activityCommentFilter %}
 {% from "govuk/components/button/macro.njk" import govukButton %}
-
+{% from "moj/components/banner/macro.njk" import mojBanner %}
 
 {% extends "case/case.njk" %}
 
 {% block sub_content %}
+  {% if hasAmendmentInProgress === true %}
+    {% set bannerHtml %}
+      <p class="govuk-body" data-cy="amendment--in-progress-bar">Amendment in progress for
+      {% for amendment in amendmentsInProgress %}
+        <a href="/case/{{dealId}}/facility/{{amendment.facilityId}}#amendments" data-cy="amendment--in-progress-bar-link"> facility {{amendment.ukefFacilityId}} </a>
+        {% if loop.index < amendmentsInProgress.length %}
```

Comment:
Validation to check whether `amendmentsInProgress` is not empty.

Reply:
done